cmd/state-svc/test/integration/svc_int_test.go

```diff
@@ -207,40 +208,61 @@ func (suite *SvcIntegrationTestSuite) TestAutostartConfigEnableDisable() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
+	autostartDir := filepath.Join(ts.Dirs.Config, "autostart")
+	err := fileutils.Mkdir(autostartDir)
+	suite.Require().NoError(err)
+	err = os.Setenv("_TEST_AUTOSTART_DIR", autostartDir)
+	suite.Require().NoError(err)
+	defer os.Unsetenv("_TEST_AUTOSTART_DIR")
+
+	if runtime.GOOS == "linux" {
+		err = fileutils.Touch(filepath.Join(autostartDir, ".profile"))
```

Comment:
Do we not write to .profile if it doesn't exist? That seems like a bug and not something we should work around in the test.

Reply:
We create the file if it doesn't exist so this isn't necessary.